ee/fleetd-chrome/package.json

```diff
@@ -17,7 +17,6 @@
     "jest": "^29.5.0",
     "jest-environment-jsdom": "^29.7.0",
     "msw": "^1.1.0",
-    "node-sass": "^8.0.0",
```

Comment:
This wasn't being used, but caused compilation failures.  I tried replacing it with `sass` as we did on the main codebase, but then it failed a dependency review which was definitely not worth working through since we're not using it anyway!

Reply:
Anyway we can know what else is not being used? Just curious.